### DISCUSSION TITLE
How can I expand the margins for one revealjs slide?

### DISCUSSION THREAD

**Discussion by eteitelbaum** [quarto-dev/quarto-cli#6595] [Q&A]
### Description

I have  a slide where I have two columns. First (left) column is text and second (right) column contains an image. The image looks too small in the column but then when I try to resize it, the image overflows the left margin with the text. 

Is there a way to resize the margins of the slide so that the text column expands out more to the left and the image column expands more out to the right? 

I tried something like [this](https://github.com/quarto-dev/quarto-cli/discussions/2337) but it did not change the size of whatever margin I specified. Not sure if this is because I have the name of the slide wrong (how can I know that for sure?) or if it just doesn't work when you have columns in the slide. 

Thanks!

**Comment by mcanouil** [comment]
Could you share a small self-contained "working" (reproducible) example to work with, _i.e._, a complete Quarto document or a Git repository? Thanks.

You can share a Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
---

This is a reproducible Quarto document using `format: html`.
It is written in Markdown and contains embedded R code.
When you run the code, it will produce a plot.

```{r}
plot(cars)
```

The end.
````
`````
